addons/website_forum/views/website_forum.xml

```diff
@@ -1486,7 +1486,7 @@
                                         <t t-set="inDropdown" t-value="True"/>
                                         <t t-set="icon" t-value="'fa-trash-o text-muted'"/>
                                         <t t-set="classes" t-value="'comment_delete'"/>
-                                        <t t-set="karma" t-value="not object.can_unlink and object.karma_unlink or 0"/>
+                                        <t t-set="karma" t-value="not (user._is_admin() or user.karma >= unlink_comment_required_karma) and unlink_comment_required_karma or 0"/>
```

Comment:
Additionally, other karma requirements are usually specified as:

"karma if not can_do_the_thing else 0".

Would align the styling here?